addons/mail/static/src/components/emojis_popover/emojis_popover.xml

```diff
@@ -2,9 +2,9 @@
 <templates xml:space="preserve">
 
     <t t-name="mail.EmojisPopover" owl="1">
-        <div class="o_EmojisPopover">
+        <div class="o_EmojisPopover" role="menu">
             <t t-foreach="emojis" t-as="emoji" t-key="emoji.unicode">
-                <span class="o_EmojisPopover_emoji" t-on-click="_onClickEmoji" t-att-title="emoji.description" t-att-data-source="emoji.sources[0]" t-att-data-unicode="emoji.unicode">
+                <span class="o_EmojisPopover_emoji" role="menuitem" t-on-click="_onClickEmoji" t-att-title="emoji.description" t-att-aria-label="emoji.description" t-att-data-source="emoji.sources[0]" t-att-data-unicode="emoji.unicode">
```

Comment:
Just saying because we might actually not want them, but those are currently not focusable I believe, this looks inconsistent with the other changes in the PR.

Reply:
Keyboard support needs to be added on a separate PR (maybe in master). Since this change adds currently little value, I'm removing it, this file is not touched anymore.